src/js/components/CookieGuard.tsx

```diff
@@ -0,0 +1,113 @@
+import { FC, Fragment, useState } from 'react';
+import { Dialog, Disclosure, Transition } from '@headlessui/react';
```

Comment:
Wel een goeie dat je hier een headless ui systeem voor gebruikt, dat is voor de toegankelijkheid ook wel fijn.
Even op letten dat je niet teveel binnenhaalt omdat het wel een library blijft maar zie dat je een analyzer hebt toegevoegd dus daar ben je wel op aan het letten verwacht ik :)